tests/test_s3blobstore.py

```diff
@@ -1,15 +1,25 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+import io
 import os
 import sys
+import time
 import unittest
 import uuid
+import select
+import boto3
+import botocore
+import contextlib
+import socket
+from multiprocessing import Process, Manager
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from botocore.vendored.requests.exceptions import ReadTimeout
```

Comment:
Following PEP8 conventions for imports, this should be on a separate block (since it's third-party)

Reply:
fixed.